source/common/ur_util.h

```diff
@@ -76,4 +78,72 @@ inline bool getenv_tobool( const char* name )
     return 0 == strcmp( "1", env );
 }
 
+inline void parse_log_env_var(const char *env_var_name,
+                              std::pair<std::string, std::string> &kv,
+                              std::string logger_name) {
+    // example: UR_LOG_LEVELS="loader=info;adapters=warn"
+    const char* env_var_cstr = std::getenv(env_var_name);
```

Comment:
It would be nice to implement this string parsing as some generic helper utility (not in this PR probably). It would be useful for parsing other env variables as well, for example: https://github.com/intel/llvm/blob/2c7ed980a43c4de633a6c32e7902116800a7876c/sycl/plugins/unified_runtime/ur/usm_allocator.cpp#L150